src/chrome/content/rules/Indymedia.xml

```diff
@@ -140,14 +131,12 @@ Fetch error: http://www.indymedia.org/ => https://www.indymedia.org/: (7, 'Faile
 	* Secured by us, see https://www.paulirish.com/2010/the-protocol-relative-url/
 
 -->
-<ruleset name="Indymedia.org (partial)" default_off='failed ruleset test'>
+<ruleset name="Indymedia.org (partial)">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="indymedia.org" />
 	<target host="athens.indymedia.org" />
-	<target host="belarus.indymedia.org" />
-	<target host="www.belarus.indymedia.org" />
 	<target host="belgium.indymedia.org" />
 	<target host="*.belgium.indymedia.org" />
```

Comment:
Can you give me an example of what you're talking about.  How would I know what URLs to test?